logger/main.go

```diff
@@ -76,7 +76,7 @@ func main() {
 			if resp != nil && resp.Node != nil {
 				drainChan <- resp.Node.Value
 			}
-			time.Sleep(time.Duration(publishInterval))
+			time.Sleep(time.Duration(publishInterval) * time.Seconds)
```

Comment:
time.Sleep(publishInterval * time.Seconds) is enough, no need to use time.Duration